src/binary.ts

```diff
@@ -292,7 +302,13 @@ export class Binary extends BSONValue {
   }
 
   inspect(): string {
-    return `new Binary(Buffer.from("${ByteUtils.toHex(this.buffer)}", "hex"), ${this.sub_type})`;
+    if (this.position === 0) {
+      return this.sub_type !== Binary.BSON_BINARY_SUBTYPE_DEFAULT
+        ? `new Binary(undefined, ${this.sub_type})`
+        : 'new Binary()';
```

Comment:
I’m a bit surprised we’re printing this – is there a meaningful difference between an “empty” `Binary` object vs. a zero-length `Binary` object? I’d have expected `new Binary()` to result in something that inspects as `Binary.createFromBase64(""[, subtype])`

Reply:
Good catch, yea as long as the position is set to zero then it doesn't matter if we've constructed the Binary with the default space or not. Better to be consistent :)